app/src/components/loading/SkeletonLoaders.tsx

```diff
@@ -0,0 +1,94 @@
+import Box from '@mui/material/Box';
+import { grey } from '@mui/material/colors';
+import Fade from '@mui/material/Fade';
+import Paper from '@mui/material/Paper';
+import Skeleton from '@mui/material/Skeleton';
+import { v4 } from 'uuid';
+
+export interface IMultipleSkeletonProps {
+  isLoading: boolean;
+  timeout?: number;
+  numberOfLines?: number;
+}
+
+const SkeletonList = (props: IMultipleSkeletonProps) => {
+  return (
+    <Fade in={props.isLoading} timeout={props.timeout ?? 1000}>
+      <Box
+        sx={{
+          position: 'absolute',
+          width: '100%',
+          height: '100%',
+          p: 1,
+          background: grey[100],
+          zIndex: 2
+        }}>
+        <Paper elevation={0} sx={{ overflow: 'hidden' }}>
+          {/* create an array of X items to build multiple skeleton components*/}
+          {Array(props.numberOfLines ?? 10)
```

Comment:
I think Jer wants the usual number of rows in a skeleton layout to be in the neighborhood of 4. @jeznorth